cli/ts/utils/constants.ts

```diff
@@ -8,3 +8,27 @@ export const oldContractAddressStoreName = "contractAddresses.old.json";
 export const contractAddressesStore = path.resolve(__dirname, "..", "..", contractAddressStoreName);
 // local file path where we are storing a previous deployment's contract addresses
 export const oldContractAddressesStore = path.resolve(__dirname, "..", "..", oldContractAddressStoreName);
+// local file name in subgraph module where we are storing the MACI contract address and block number
+export const subgraphNetworkStoreName = "networks.json";
+// local file name in subgraph module where we are storing the previous deployment's MACI contract address and block number
+export const oldSubgraphNetworkStoreName = "networks.old.json";
```

Comment:
I don't think we need to keep all the previous deploy info.

Reply:
okay, so we don't need to backup the previous deployment